roles/contiv/templates/contiv.cfg.j2

```diff
@@ -0,0 +1,6 @@
+{
+  "K8S_API_SERVER": "https://{{ hostvars[groups['masters'][0]]['ansible_' + netmaster_interface].ipv4.address }}:{{ kube_master_api_port }}",
```

Comment:
One of the current suggested solutions for master HA is to use keepalived, ucarp or equivalent solution to move the first master'sapi server IP address to a second master if the first master goes down. This allows the solution to be independent of needing to access the cluster hostname through skydns (which itself may not yet be up during initial stages of contiv networking bringup).